plugins/actix-web/src/web.rs

```diff
@@ -521,6 +517,10 @@ impl ServiceFactory<ServiceRequest> for Route {
     type InitError = ();
     type Service = <actix_web::Route as ServiceFactory<ServiceRequest>>::Service;
     type Future = <actix_web::Route as ServiceFactory<ServiceRequest>>::Future;
+    // type Response =
```

Comment:
Do we need this as well?

Reply:
Ah yes, this is the removal of the hardcoded reference to ServiceResponse I mentioned. Forgot to comment that back in.